### DISCUSSION TITLE
Volume usage without collection

### DISCUSSION THREAD

**Discussion by Koyorice** [seaweedfs/seaweedfs#6306] [Q&A]
When I use s3 to upload files, volume will automatically create 14 volumes
7 of them have collection but the other 7 don't, I want to know what is the use of these 7 volume without collection

![image](https://github.com/user-attachments/assets/78082548-8a53-4614-b57e-a58cc94f335c)

Thanks!

**Comment by chrislusf** [comment]
"these 7 volume without collection" they are default collections, used for system logs.

**Reply by Koyorice** [reply]
Can I change the number of volumes it uses or disable system log?
One of my volumes has 100G, I think this wastes a lot of free space

**Reply by chrislusf** [reply]
you can delete the logs under `/topics/.system/...` by date.

**Reply by Koyorice** [reply]
But this way it will still use 7 volumes
I hope to use only one volume for the log, 100G is enough
